src/components/EditClimbForm.js

```diff
@@ -16,21 +17,21 @@ function EditClimbForm(props) {
         <div >Rope: {climb.rope}</div>
         <div >Colour: {climb.colour}</div>
         <div >Grade:
-            <input value={grade} onChange={(e) => {
+            <TextField id="outlined-basic" variant="outlined" value={grade} onChange={(e) => {
```

Comment:
Not really an appropriate ID. If this is meant to be CSS that can be reproduced across any "outline basic" text box, use a class instead. If it's meant to only be an ID then use something that describes the _identifies_ the element rather than describing what it should look like, e.g. "grade-input".

Reply:
Removed ID